src/Symfony/Bundle/FrameworkBundle/Secrets/SodiumVault.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Secrets;
+
+/**
+ * @author Tobias Schultze <http://tobion.de>
+ * @author Jérémy Derussé <jeremy@derusse.com>
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+class SodiumVault
+{
+    private $encryptionKey;
+    private $decryptionKey;
+    private $pathPrefix;
+
+    public function __construct(string $pathPrefix)
```

Comment:
I would revert the name of the constructor argument back to `$secretsDir`, as that's actually what is expected here.

Only the class property needed a renaming, because it does not store the dir itself, but appends a filename prefix too.

Reply:
reverted